src/gigs-board/pages/admin.jsx

```diff
@@ -0,0 +1,264 @@
+/* INCLUDE: "common.jsx" */
```

Comment:
Can we isolate this PR to only be adding the new admin page? And can we have this admin page be rebased to use the new refactor/root, please?

Reply:
Okay, I'll create a separate PR for UIkit adjustments soon, I needed to make some of the components more flexible